packages/lit-protocol-cipher/src/lit-protocol-cipher-provider.ts

```diff
@@ -147,8 +154,9 @@ export default class LitProtocolCipherProvider implements CipherProviderTypes.IC
 
       // Create the SIWE message
       const toSign = await createSiweMessage({
-        domain,
-        statement,
+        // Use class properties instead of function parameters
+        domain: this.domain,
+        statement: this.statement,
```

Comment:
Since we use `this` inside the callback, should we `bind()` the callback to ensure it always uses the current `this` and not some other `this`?

```ts
const authNeededCallback = async (params: AuthCallbackParams) => {
...
  domain: this.domain,
  statement: this.statement,
...
}.bind(this); // <-- should we add this line?
```